editors/code/src/extension.ts

```diff
@@ -632,7 +242,10 @@ export async function activate(context: vscode.ExtensionContext) {
               vscode.commands.executeCommand("luau-lsp.reloadServer");
             }
           });
-      } else if (e.affectsConfiguration("luau-lsp.types")) {
+      } else if (
+        e.affectsConfiguration("luau-lsp.types") ||
+        e.affectsConfiguration("luau-lsp.platform")
```

Comment:
I think platform here might be too broad - we don't need to reload the whole server if a user decides to enable / disable the Studio plugin, or configure the way the sourcemap works.